### ISSUE TITLE
Application of machine learning techniques

### ISSUE THREAD

**Issue by dolejarz** [CityofToronto/bdit_incidents#9]
Tentative game plan:
- Create a table of 90% of rows that have a MIR field
- Train a decision tree classifier using nLanes, incType, and incDuration as features, MIR as labels
- Test accuracy against the remaining 10% of rows
- If accuracy is good, go ahead and label the rest of the rows that don't have a MIR field

**Comment by dolejarz** [comment]
[using pd.df in sklearn](https://www.quora.com/How-does-python-pandas-go-along-with-scikit-learn-library)
[remember to cast strings to int/float!!!](http://manishamde.github.io/blog/2013/03/07/pandas-and-python-top-10/#numpy)

**Comment by dolejarz** [comment]
[use pickle](https://wiki.python.org/moin/UsingPickle) to pass numpy arrays between preprocessing script and machine learning script

**Comment by dolejarz** [comment]
Tweak parameters of decision tree classifier to get higher precision and recall

**Comment by dolejarz** [comment]
use sklearn's [one hot encoder](http://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.OneHotEncoder.html#sklearn.preprocessing.OneHotEncoder) in preprocessing. Use [label encoder](http://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.LabelEncoder.html#sklearn.preprocessing.LabelEncoder) for non numerical values
